tools/sdk-cli/src/commands.rs

```diff
@@ -80,6 +86,8 @@ pub(crate) enum Commands {
         sat_per_vbyte: u32,
     },
 
+    MaxOnchainPayment {},
```

Comment:
Any reason this is named differently in the cli? I could read this as max send or receive onchain payment

Reply:
Leftover, Fixed.